openedx/features/marketplace/migrations/0001_initial.py

```diff
@@ -29,8 +28,8 @@ class Migration(migrations.Migration):
                 ('country', django_countries.fields.CountryField(max_length=2)),
                 ('city', models.CharField(max_length=255)),
                 ('video_link', models.URLField(blank=True, null=True)),
-                ('image', models.ImageField(blank=True, help_text='Accepted extensions: .jpg, .png', null=True, upload_to=functools.partial(openedx.features.idea.helpers.upload_to_path, *(), **{b'folder': 'images'}), validators=[django.core.validators.FileExtensionValidator(['jpg', 'png'])])),
-                ('file', models.FileField(blank=True, help_text='Accepted extensions: .docx, .pdf, .txt', null=True, upload_to=functools.partial(openedx.features.idea.helpers.upload_to_path, *(), **{b'folder': 'files'}), validators=[django.core.validators.FileExtensionValidator(['docx', 'pdf', 'txt'])])),
+                ('image', models.ImageField(blank=True, help_text='Accepted extensions: .jpg, .png', null=True, validators=[django.core.validators.FileExtensionValidator(['jpg', 'png'])])),
+                ('file', models.FileField(blank=True, help_text='Accepted extensions: .docx, .pdf, .txt', null=True, validators=[django.core.validators.FileExtensionValidator(['docx', 'pdf', 'txt'])])),
```

Comment:
why are you updating the migrations??

Reply:
I have added a note in the [description](https://github.com/philanthropy-u/edx-platform/pull/1545#issue-418461889), regarding this.